udata/core/dataset/models.py

```diff
@@ -87,6 +88,10 @@
 PIVOTAL_DATA = 'pivotal-data'
 CLOSED_FORMATS = ('pdf', 'doc', 'word', 'xls', 'excel')
 
+# Maximum acceptable Damerau-Levenshtein distance
```

Comment:
No, it's the number of operations on a single string to go to another.
It's not related to the length but I think you refer to elasticsearch which is doing progressive Levenshtein distance: do not allow typo on low string, allow 1 on medium string and 2 on large string. In our case the corpus is known, they all have title with a significant length to allow 2 typos and not to be close of another license name.